src/ui/components/Navigation/Core/TableView/TableView.scss

```diff
@@ -249,6 +249,33 @@
         }
     }
 
+    &__edit-favorites-alias-btn {
+        visibility: hidden;
+        margin-right: 5px;
```

Comment:
Can we use here only `visibility` prop and move the rest of the props to `FavoritesNameWithAliasItem.scss`?
Looks like styling a child component from its parent. Not critical, final decision up to you.

Reply:
Yes, agree, refactored